launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py

```diff
@@ -296,7 +296,10 @@ def create_time_series_outlier_filter_components(input_topic, output_topic):
                 plugin="occupancy_grid_map_outlier_filter::OccupancyGridMapOutlierFilterComponent",
                 name="occupancy_grid_map_outlier_filter",
                 remappings=[
-                    ("~/input/occupancy_grid_map", "/perception/occupancy_grid_map/map"),
+                    (
+                        "~/input/occupancy_grid_map",
+                        "/perception/occupancy_grid_map/map",
```

Comment:
I think your formatter change automatically. need to change? :pray:

Reply:
fix